modules/update-other-images-40.adoc

```diff
@@ -79,6 +79,11 @@ Or if you're using collector slim:
 ----
 $ oc -n stackrox set image ds/collector collector=registry.redhat.io/advanced-cluster-security/rhacs-collector-slim-rhel8:{rhacs-version}
 ----
+. Update the Admission Control image:
```

Comment:
I _think_ that, when deploying, you can opt out from deploying admission control. In this case the command will fail.
I'm checking that now.